### PR TITLE
Release/3.0.0

### PR DESCRIPTION
**Pull request by FlorianBouron**
## Bump packages for the release 3.0.0

Commits:

- [Bump packages for the release 3.0.0](https://github.com/GemWallet/gemwallet-extension/pull/167/commits/24174b5e36422107dd965d306c3d902f1beb6cd5)
- [Update the releaseNpm script](https://github.com/GemWallet/gemwallet-extension/pull/167/commits/465924d146104abf732b8e34851da5c130ef7a3d)
- [Add request types](https://github.com/GemWallet/gemwallet-extension/pull/167/commits/a20b2d0c02d82e04b893b74aae0a54fe74c4ebee)

### CODE REVIEW

File: packages/api/src/index.ts [hunk 1]
```diff
@@ -16,7 +16,16 @@ export type {
   TrustSetFlags
 } from '@gemwallet/constants';
 
-// API return types
+// API request types
+export type {
+  GetNetworkRequest,
```
Comment (FlorianBouron) [inline packages/api/src/index.ts:21]: @florent-uzio do you think these types should be renamed like that:
```suggestion
  GetNetworkRequest as GetNetworkParameter,
```

File: packages/api/src/index.ts [hunk 2]
```diff
@@ -16,7 +16,16 @@ export type {
...
+  GetNFTRequest,
+  SendPaymentRequest,
+  SetTrustlineRequest,
+  SignMessageRequest
+} from '@gemwallet/constants';
```
Comment (florent-uzio) [inline packages/api/src/index.ts:26]: Should all those types come from a "models" directory?

**Review by ThibautBremand — APPROVED** [review]

File: packages/api/src/index.ts (hunk 2 above)
Comment (FlorianBouron) [reply packages/api/src/index.ts:26]: We are gonna move them after we release the v3 :) 
Thank you for the feedback.

File: dist/gemwallet-api.js [hunk 3]
```diff
@@ -1 +1 @@
...
..."Testnet",e.DEVNET="Devnet",e.AMM_DEVNET="AMM-Devnet"}(r=t.Network||(t.Network={})),function(e){e.MAINNET="wss://xrplcluster.com",e.TESTNET="wss://s.altnet.rippletest.net:51233",e.DEVNET="wss://s.devnet.rippletest.net:51233",e.AMM_DEVNET="wss://amm.devnet.rippletest.net:51233"}(o=t.NetworkServer||(t.NetworkServer={})),t.NETWORK=((n={})[r.MAINNET]={name:r.MAINNET,server:o.MAINNET,description:"Main network using the production version of the XRP Ledger."},n[r.TESTNET]={name:r.TESTNET,server:o.TESTNET,description:"Acts as a testing network, without impacting production users and risking real money."},n[r.DEVNET]={name:r.DEVNET,server:o.DEVNET,description:"A preview of upcoming features, where unstable changes are tested out."},n[r.AMM_DEVNET]={name:r.AMM_DEVNET,server:o.AMM_DEVNET,description:"XLS-30d Automated Market Makers preview network."},n)},726:(e,t)=>{Object.defineProperty(t,"__esModule",{value:!0})},780:(e,t)=>{Object.defineProperty(t,"__esModule",{value:!0})},875:(e,t)=>{Object.defineProperty(t,"__esModule",{value:!0})}},t={};return function n(r){var o=t[r];if(void 0!==o)return o.exports;var i=t[r]={exports:{}};return e[r].call(i.exports,i,i.exports,n),i.exports}(607)})()));
```
Comment (florent-uzio) [inline dist/gemwallet-api.js:1]: Does `dist` directory need to be pushed?

Comment (FlorianBouron) [reply dist/gemwallet-api.js:1]: Yes some people are consuming it as a CDN

**Pull request merged**